test-data/unit/check-unions.test

```diff
@@ -556,7 +556,7 @@ reveal_type(d1)  # N: Revealed type is "Union[Any, builtins.float]"
 reveal_type(d2)  # N: Revealed type is "Union[Any, builtins.float]"
 
 e: Union[Any, Tuple[float, float], int]
-(e1, e2) = e  # E: "builtins.int" object is not iterable
```

Comment:
There are a lot of places where `builtins.` is used, and some places where it is not used. I cannot tell what kind of principle is used here 😞

Reply:
`mypy` keeps the prefix in `reveal_type` for more verbose information. And according to the original discussion, it's better to discard `builtins.` to get concise error messages. Also, cpython dosen't output `builtins.` when error occurs.